server/middleware/setUpWebSecurity.ts

```diff
@@ -29,6 +29,7 @@ export default function setUpWebSecurity(): Router {
           imgSrc: ["'self'", 'https://www.google-analytics.com'],
         },
       },
+      crossOriginEmbedderPolicy: true,
```

Comment:
without this, remote jquery ui components do not load properly. i don't think this is the right fix, but it's what other projects are doing.

1) i don't think jquery ui is in fact used anywher
2) the proper solution is to load all front-end assets from the same origin

Reply:
I thought jQuery UI was used by some of the govuk components, e.g. tabs? But I tried to comment the import and things seem to work fine?

1. If jQuery UI is not used, shouldn't we remove the import maybe?
2. true, there is a `jquery-ui-dist` node package or maybe we just download the files...

However, if we can be sure jQuery UI is not used probably remove is the best.

Reply:
GDS design system does not use jquery or jquery ui. MoJ design system _might_ but i don't think it does.